main.java

```diff
@@ -14,7 +14,7 @@
 //DEPS org.openjfx:javafx-fxml:18-ea+9:${os.detected.jfxname}
 
 // actual jar we want to run
-//DEPS com.github.maxandersen:jfxcentral:jbang-SNAPSHOT
```

Comment:
note: this is using `-SNAPSHOT` so it will get the latest which might not be what you want as it will make new launches slow. Might want to pick a more stable sha,tag or branch so it does not impact everyone on every build.

Reply:
ok